go.mod

```diff
@@ -87,26 +87,26 @@ require (
 	golang.org/x/sync v0.10.0
 	golang.org/x/sys v0.28.0
 	golang.org/x/text v0.21.0
-	golang.org/x/time v0.8.0
-	golang.zx2c4.com/wireguard/wgctrl v0.0.0-20200324154536-ceff61240acf
+	golang.org/x/time v0.9.0
+	golang.zx2c4.com/wireguard/wgctrl v0.0.0-20241231184526-a9ab2273dd10
```

Comment:
@tomastigera Any concerns to bump the wgctrl package? This aligns with what enterprise is using.